standards/collections.md

```diff
@@ -69,6 +70,7 @@ POST https://api.spscommerce.com/v1/books
      &offset=int                               
      &cursor=string
 // REQUEST
+HTTP/1.1 200
```

Comment:
Shouldn't this be 
```
POST https://api.spscommerce.com/v1/books
{
  "paging": {
  ...
}

Reply:
ah looks like I copy pasted into the request thinking it was a response. Thanks!